### PR TITLE
FLOE-160: Namespace change and directory restructure.

### PR DESCRIPTION
**Pull request by cindyli**
This pull request does,
1. Change the namespace for metadata components to gpii.metadata;
2. Re-construct directory structures to relocate the files for metadata and feedback demos;
3. Use css to implement the blank box used on the metadata landing page for "create new resource" link, rather than using an image.

http://issues.fluidproject.org/browse/FLOE-160

### CODE REVIEW

File: tests/components/metadata/js/audioPanelTests.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ https://github.com/gpii/universal/LICENSE.txt
     fluid.registerNamespace("fluid.tests");
 
     fluid.tests.checkAudioState = function (audioPanel, expectedRadiobuttons, expectedCheckboxes, state) {
-        audioPanel = audioPanel.typeName === "fluid.metadata.audioPanel" ? audioPanel : audioPanel;
+        audioPanel = audioPanel.typeName === "gpii.metadata.audioPanel" ? audioPanel : audioPanel;
```
Comment (amb26) [inline tests/components/metadata/js/audioPanelTests.js]: As written, this code is a no-op

Comment (cindyli) [inline tests/components/metadata/js/audioPanelTests.js]: Good catch, @amb26. Fixed.

File: tests/components/metadata/html/resourceInputPanel.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                        url: "../../src/html/resourceInput-template.html"
+                    }
+                },
+                // listeners: {
+                //     onReady: function (that) {
+                //         that.applier.change("audio", "available");
+                //         that.applier.change("dialogue", true);
+                //         that.applier.change("audio", "unavailable");
+                //     }
+                // }
```
Comment (jobara) [inline tests/components/metadata/html/resourceInputPanel.html]: Should this commented out listener be removed?

File: tests/components/metadata/js/metadataTestsCommon.js [hunk 3]
```diff
@@ -22,14 +22,14 @@ https://github.com/gpii/universal/LICENSE.txt
     };
 
     fluid.tests.checkRenderedVideoMetadataPanel = function (videoMetadataPanel) {
```
Comment (jobara) [inline tests/components/metadata/js/metadataTestsCommon.js]: we should probably change these to the gpii namespace too

**Comment by cindyli** [comment]
All addressed, @jobara. Also made more namespace changes that were missed earlier but uncovered by your hint on "fluid.tests".

**Comment by acheetham** [comment]
@cindyli, since we're rearranging the folder hierarchy anyway, what do you think about moving the main demo index.html files out of the html folder and up into the particular demo folders, so that the urls for the demos don't have to include "/html" at the end?

**Comment by cindyli** [comment]
Makes sense, @acheetham. Made the change.

**Comment by jobara** [comment]
[floe-foundation-readme.txt](https://github.com/cindyli/metadata/blob/FLOE-160/demos/lib/floe-foundation-readme.txt) should probably be converted to a VERSION.md file and moved into the foundation directory.

**Comment by jobara** [comment]
@cindyli could you also create a pull request against build.fluidproject.org to update the url to the metadata demo as needed.

**Comment by cindyli** [comment]
@jobara, a pull request has been issued against build.fluidproject.org to update the link to the metadata authoring demo and add a link to the metadata feedback demo: https://github.com/fluid-project/build.fluidproject.org/pull/12

All comments are addressed and ready for another round.

File: tests/components/metadata/html/pouchDBDataSource-Test.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```
Comment (jobara) [inline tests/components/metadata/html/pouchDBDataSource-Test.html]: The tests for the pouchDBDataSource should be in a separate directory like the component is.

File: tests/components/metadata/all-tests.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    <meta charset="UTF-8" />
+    <title>Metadata complete tests runner</title>
+
+    <link rel="stylesheet" href="../../lib/infusion/lib/qunit/css/qunit.css" type="text/css" media="screen">
+    <link rel="stylesheet" href="../../lib/infusion/lib/qunit/addons/composite/qunit-composite.css">
+    <script src="../../lib/infusion/lib/qunit/js/qunit.js"></script>
+    <script src="../../lib/infusion/lib/qunit/addons/composite/qunit-composite.js"></script>
+
+    <script>
+    QUnit.testSuites([
```
Comment (jobara) [inline tests/components/metadata/all-tests.html]: We should name the test suite like it is done in the feedback all tests.

File: tests/all-tests.html [hunk 6]
```diff
@@ -8,24 +8,11 @@
...
-        "./html/panel-Test.html",
-        "./html/pouchDBDataSource-Test.html",
-        "./html/resourceInputPanel-Test.html",
-        "./html/transcriptsPanel-Test.html",
-        "./html/videoMetadataPanel-Test.html",
-        "./html/videoPanel-Test.html",
-        "./components/feedback/html/bindDialog-Test.html",
-        "./components/feedback/html/feedback-Test.html",
-        "./components/feedback/html/matchConfirmation-Test.html"
+        "./components/metadata/all-tests.html",
```
Comment (jobara) [inline tests/all-tests.html]: We should name the test suite like it is done in the feedback all tests. You can even put these in separate calls to `QUnit.testSuites` to give them different names.

**Comment by cindyli** [comment]
All addressed, @jobara.

**Comment by jobara** [comment]
merged at c747311198a803faa7d16a561bb7df412612b6ce

**Pull request merged**